### DISCUSSION TITLE
Workflow for testing changes

### DISCUSSION THREAD

**Discussion by JackCurragh** [skiniry/Trips-Viz#8] [General]
I'm not sure how best to go about making a change to trips and having it available for you to decide whether its appropriate etc. 

@ciarajudge whats the best way to go about this? Do we branch to address a specific idea and then that gets merged when its addressed?

Also, is this version of trips up to date now or are there changes to be pushed?

**Comment by skiniry** [comment]
I think making a branch for each specific idea is the way to go alright, and yeah everything on here is now up to date with what's on poitin.

**Reply by JackCurragh** [reply]
Just tried to make a fresh clone and it looks like celery_app is missing?

**Reply by ciarajudge** [reply]
Yeah I'm thinking if I work on the UI on it's own branch and then I can make a pull request to merge the changes into main. Best to do that way as I will be doing a lot of trial and error in terms of moving stuff around etc. Also my commit comments can sometimes be a bit stupid so better if they're not on the main branch
